x-pack/plugins/security_solution/public/resolver/view/panels/use_format_date.ts

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import moment from 'moment-timezone';
+import { useUiSetting } from '../../../../../../../src/plugins/kibana_react/public';
+import { formatter, invalidDateText } from './panel_content_utilities';
+
+export function useDateFormatSetting(
```

Comment:
The `formatDate` function in `panel_content_utilities` is also used in `node_events_of_type.tsx` and `event_detail.tsx` (via the `relatedEventDisplayInfoByEntityAndSelfId` selector). Do we not want to format those as well?